ios/FluentUI/Navigation/NavigationBar.swift

```diff
@@ -273,14 +312,17 @@ open class NavigationBar: UINavigationBar, TokenizedControlInternal {
     private let contentStackView = ContentStackView() //used to contain the various custom UI Elements
     private let rightBarButtonItemsStackView = UIStackView()
     private let leftBarButtonItemsStackView = UIStackView()
-    private let leadingSpacerView = UIView() //defines the leading space between the left and right barbuttonitems stack
+    private let preTitleSpacerView = UIView() //defines the spacing before the title, used for compact centered titles
```

Comment:
Nit: space after //. Is this something swiftlint looks for?

Reply:
> Is this something swiftlint looks for?
Apparently not, since otherwise we would've heard it complain for a long time.

But yeah, they do look better with spaces.